### ISSUE TITLE
[$500] Desktop deploys always give a "false negative"

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#35629]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @roryabraham 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1706831827428559

## Action Performed:
*Prerequisite:* You have the desktop app and there's an update available, but you haven't been notified of it yet. Note that you can download a `.dmg` for an older app version from the [GitHub Release](https://github.com/Expensify/App/releases). Then:
1. Go to the `New Expensify` menu, and click `Check for updates`

## Expected Result: 
A modal pops up that says an update is available. You can click on a button in the modal to update the app.

## Actual Result: 
A modal pops up that says you're up to date. A few moments later, you receive a notification saying that an update is available. You can click on the notification to update.

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0153d36244ff9ed9b3</li>
        <li>Upwork Job ID: 1813344570140106347</li>
        <li>Last Price Increase: 2024-09-06</li>
    </ul>
</details>

**Comment by laurenreidexpensify** [comment]
Waiting for weekly retesting for reproduction

**Comment by laurenreidexpensify** [comment]
Waiting for retesting

**Comment by laurenreidexpensify** [comment]
Asking in Slack https://expensify.slack.com/archives/C9YU7BX5M/p1708103233932099 if this was confirmed as reprocudible

**Comment by laurenreidexpensify** [comment]
For Testing h/t @roryabraham - 

> You can reproduce it on-demand by installing an older version of the desktop app. I realize that we don't leave build artifacts on the GitHub Releases, which makes it harder to get older app versions. Going to make an issue for that

**Comment by laurenreidexpensify** [comment]
Waiting for reproduction

**Comment by laurenreidexpensify** [comment]
Waiting for reproduction

**Comment by roryabraham** [comment]
@laurenreidexpensify this new feature idea would make this much easier to test: https://github.com/Expensify/App/issues/36758

**Comment by mvtglobally** [comment]
QA is unable to verify this to confirm if issue is reproducible or no. This needs a build pushed and checked which is a bit complex to catch in the right moment

**Comment by roryabraham** [comment]
ok, we'll put this on HOLD for https://github.com/Expensify/App/issues/36758, which will make this easier to test.

It's also necessary for testing https://github.com/Expensify/App/issues/37349, so I think it needs to be prioiritized soon

**Comment by laurenreidexpensify** [comment]
Still held on #36758

**Comment by mvtglobally** [comment]
It seems as this Issue is reproducible during KI retests. 
https://github.com/Expensify/App/assets/43995119/b47d33f4-c9b6-4df9-9caf-8328dc2801d6

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
Still held on https://github.com/Expensify/App/issues/36758

**Comment by laurenreidexpensify** [comment]
@kadiealexander reassigning this one while I am on parental leave. TLDR is that you just need to check in every so often and confirm that https://github.com/Expensify/App/issues/36758 is still in progress. 

It also might be worth checking in with @roryabraham if we still require this to be open, or if it's worth closing at this point. Thanks!

**Comment by kadiealexander** [comment]
@roryabraham do you think this needs to stay open?

**Comment by roryabraham** [comment]
I think so. I created https://github.com/Expensify/App/issues/36758 on Friday to start making progress on this.

**Comment by roryabraham** [comment]
Taking this off HOLD since https://github.com/Expensify/App/issues/36758 is done

**Comment by hungvu193** [comment]
Waiting for proposals.

**Comment by hungvu193** [comment]
Still waiting for proposals

**Comment by roryabraham** [comment]
There are some instructions on [testing electron auto-update](https://github.com/Expensify/App/blob/59a07d6b8ba20b13cdaf5e3dc388a8bc8f33f61e/desktop/README.md#testing-electron-auto-update), though I'm not sure how up-to-date they are. They also require a paid apple developer account.

**Comment by hungvu193** [comment]
> There are some instructions on [testing electron auto-update](https://github.com/Expensify/App/blob/59a07d6b8ba20b13cdaf5e3dc388a8bc8f33f61e/desktop/README.md#testing-electron-auto-update), though I'm not sure how up-to-date they are. They also require a paid apple developer account.

Can you ask someone internal if these docs are up to date, I have a paid apple account so I can give it a try

**Comment by roryabraham** [comment]
> Can you ask someone internal if these docs are up to date

I don't think anyone will know the answer. Best way to find out is to try

**Comment by hungvu193** [comment]
Cool. I'm gonna try it this weekend.

**Comment by hungvu193** [comment]
Not overdue. Still trying to setup Minio

**Comment by hungvu193** [comment]
Little update, I can setup Minio locally, but when I tried to push a build to Min.IO, I got this error below even though I had `Developer ID Application` certificate installed on my laptop.
 
<img width="1046" alt="Screenshot 2024-08-08 at 09 30 55" src="https://github.com/user-attachments/assets/adc91bcb-be41-490d-9c66-eec22d63ca89">

**Comment by hungvu193** [comment]
Still no luck. This probably needs an internal engineer.

**Comment by hungvu193** [comment]
Same here

> Still no luck. This probably needs an internal engineer.

**Comment by hungvu193** [comment]
No update

**Comment by kadiealexander** [comment]
@roryabraham do you think this is still an issue?

**Comment by kadiealexander** [comment]
@roryabraham do you think this is still an issue?

**Comment by roryabraham** [comment]
well now we can have Applause retest this one. requested retest: https://expensify.slack.com/archives/C9YU7BX5M/p1724907395238649

also, don't think it needs to be daily

**Comment by roryabraham** [comment]
bumped applause in slack

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by kadiealexander** [comment]
@roryabraham should we close this one?

**Comment by roryabraham** [comment]
Yep, let's close it out since it's no longer reproducible
